lib/sudo-exec/src/event.rs

```diff
@@ -9,19 +19,28 @@ pub(crate) type Callback<T> = fn(&mut T, &mut EventHandler<T>);
 
 /// A type able to poll events from file descriptors and run callbacks when events are ready.
 pub(crate) struct EventHandler<T> {
+    signal_handlers: HashMap<c_int, SignalHandler>,
```

Comment:
for readability, could you type alias this `c_int` to something like `SignalType` (or `SignalKind`)